data_model/src/lib.rs

```diff
@@ -84,11 +84,13 @@ impl ValidationError {
     DebugCustom, Display, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Encode, Serialize, IntoSchema,
 )]
 #[repr(transparent)]
-pub struct Name(String);
+pub struct Name(ConstString);
 
 impl Name {
+    const EMPTY: Self = Self(ConstString::new());
+
     pub(crate) const fn empty() -> Self {
```

Comment:
it's a const function, no need for `const EMPTY`

Reply:
oh, okay, it was `const` function from the begging.

Reply:
you suggested introduce shared const value for empty names

Reply:
I remember suggesting that somewhere. If it was here, it was a bad suggestion

Reply:
ok, i will return to initial implementation)